src/gui/categoryfilterwidget.h

```diff
@@ -26,7 +26,23 @@
  * exception statement from your version.
  */
 
- #include <QTreeView>
+#include <QSortFilterProxyModel>
+#include <QTreeView>
+
+class CategoryFilterProxyModel: public QSortFilterProxyModel
+{
+    Q_OBJECT
```

Comment:
He probably copied from our other proxymodels. At least `TransferListSortModel` has this.
FYI, I would prefer the declaration stay in a header file(possibly its own) instead of a cpp file.